solutions/node-hello-world/README.md

```diff
@@ -0,0 +1,31 @@
+# Node.js Hello World
```

Comment:
Just double checking: Is this here on purpose? Didn’t see it mentioned in the pr description